cli/enry/main.go

```diff
@@ -77,8 +80,17 @@ func main() {
 		log.Fatal(err)
 	}
 
-	data, _ := json.MarshalIndent(out, "", "  ")
-	fmt.Printf("%s\n", data)
+	if *jsonFlag && !*breakdownFlag {
+		printJson(out)
+	} else if *jsonFlag && *breakdownFlag {
+		printBreakDown(out)
+	} else if *breakdownFlag {
+		printPercents(out)
+		fmt.Println()
```

Comment:
maybe is better to use `bytes.Buffer` to generate the complete string, and then print it to stdout instead of use `fmt.Println()` here and into `printPercents` and `printBreakDown` functions.